packages/mobile/src/screens/upload-screen/screens/UploadingTracksScreen.tsx

```diff
@@ -65,25 +65,7 @@ export const UploadingTracksScreen = () => {
   const dispatch = useDispatch()
 
   useEffectOnce(() => {
-    // set download gate based on stream gate
-    // this will be updated once the UI for download gated tracks is implemented
-    const tracksToUpload = tracks.map((track) => {
-      const isDownloadable = !!track.metadata.download?.is_downloadable
-      const isDownloadGated = track.metadata.is_stream_gated
-      const downloadConditions = track.metadata.stream_conditions
-      return {
-        ...track,
-        metadata: {
-          ...track.metadata,
-          is_downloadable: isDownloadable,
-          is_download_gated: isDownloadGated,
-          download_conditions: downloadConditions
-        }
-      }
-    })
-    dispatch(
-      uploadTracks(tracksToUpload, undefined, UploadType.INDIVIDUAL_TRACK)
-    )
+    dispatch(uploadTracks(tracks, undefined, UploadType.INDIVIDUAL_TRACK))
```

Comment:
removed the logic from here and put in in EditTrackScreen above (with additional handling of lossless etc.) so that it works for both upload and edit.

Reply:
i don't see where the upload flow would go through the logic in EditTrackScreen above, just making sure we're not skipping it. don't see a submit button in UploadingTracksScreen.